addons/l10n_mx_edi/data/templates/mx_invoice.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <template id="mx_invoice">
+            <cfdi__Comprobante
+                t-att="{
+                    'xsi:schemaLocation': 'http://www.sat.gob.mx/cfd/3 http://www.sat.gob.mx/sitio_internet/cfd/3/cfdv32.xsd',
+                    'xmlns:cfdi': 'http://www.sat.gob.mx/cfd/3',
+                    'xmlns:xsi': 'http://www.w3.org/2001/XMLSchema-instance'
+                    }"
+                version="3.2"
+                sello=""
+                t-att-folio="number"
+                t-att-fecha="date"
+                t-att-certificado="certificate"
+                t-att-formaDePago="payment_policy"
+                t-att-noCertificado="certificate_number"
+                t-att-subTotal="amount_untaxed"
+                t-att-Moneda="currency_name"
+                t-att-total="amount_total"
+                t-att-metodoDePago="pay_method"
+                t-att-tipoDeComprobante="document_type"
+                t-att-LugarExpedicion="domicile">
+                <cfdi__Emisor
+                    t-att-rfc="rfc(supplier) if supplier.vat else None"
```

Comment:
In our experience this logic should not be into the template, this is why we created a method for such decisions, the template should have as less logic as possible in order to be able to test it cleaner.

In fact what we were planning to do is:

PAss all parameters directly to the template and then post-process the xml-remove all empty attrs AND after that pass the validation, on this way this will return always errors with only the required values.